src/channel.ts

```diff
@@ -2,6 +2,27 @@ import { api } from '@replit/protocol';
 import type { ChannelCloseReason, RequestResult } from './types';
 import CrosisError from './util/CrosisError';
 
+type CustomFallbackBehavior = (
+  cmds: api.ICommand[],
+  failedIndex: number,
+  successfulResults: RequestResult[],
+  failureResult: RequestResult,
```

Comment:
```suggestion
  failedResult: RequestResult,
```

keep grammar consistent :)

Reply:
Wouldn't I need to change it to succeededResults? I'll swap successful for success and then I think the consistency unambiguous.